src/features/content-explorer/content-explorer/ContentExplorer.js

```diff
@@ -395,9 +403,20 @@ class ContentExplorer extends Component {
 
     renderItemListEmptyState = () => {
         const { foldersPath, isInSearchMode } = this.state;
+        const { isIncludeSubfoldersAllowed } = this.props;
         const isViewingSearchResults = isInSearchMode && foldersPath.length === 1;
+        const isOnInitialModalPage =
+            foldersPath.length === 1 &&
+            foldersPath[0].id === INITIAL_FOLDERS_PATH[0].id &&
+            foldersPath[0].name === INITIAL_FOLDERS_PATH[0].name;
```

Comment:
do we need to check on name here too? Id is unique

Reply:
It might not be necessary. I just wanted to cover all bases to ensure it was the initial folder path